server/cli/kv.go

```diff
@@ -170,7 +219,7 @@ func runDel(cmd *commander.Command, args []string) {
 // range.
 var CmdScan = &commander.Command{
 	UsageLine: "scan [options] [<start-key> [<end-key>]]",
-	Short:     "scans a range of keys",
+	Short:     "scans a range of keys\n",
```

Comment:
It's a hack to make `cockroach help` look better by providing separators between groups of related commands.